### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by Arshavin86**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -13,9 +13,9 @@ const cors = require('cors');
 app.use(cors());
 app.options('*', cors());
 
-app.use(express.static(__dirname + '/../client/dist'));
+app.use(express.static('public'));
 
-app.get('/checkout/:id', jsonParser, (req, res) => {
+app.get('/api/checkouts/:id', jsonParser, (req, res) => {
```
Comment (hgtruong) [inline server/index.js]: The url path we agreed on is: "/api/turash/[module]/:id"

File: public/bundle.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (hgtruong) [inline public/bundle.js]: Don't upload bundle.js

File: package.json [hunk 3]
```diff
@@ -28,6 +28,7 @@
     "@babel/preset-react": "^7.0.0",
     "babel-loader": "^8.0.4",
     "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
```
Comment (hgtruong) [inline package.json]: Don't use hack-reactor style guide. Use AirBnB style guide.

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<html>
+  <head>
+  </head>
+  <body>
+        <h1>Checkout</h1>
+    <div id='root'>I am static!!</div>
```
Comment (therobinkim) [inline public/index.html]: use double quotes in HTML

File: faker.js [hunk 5]
```diff
@@ -2,25 +2,41 @@ const faker = require ('faker');
...
+      currentDate = currentDate.addDays(1);
+  }
+  return dateArray;
+}
+
+const loadNewCar = ((times=101) => {
+
+  for (let i = 1; i < times; i++) {
+    const rangeTimes = faker.random.number({
         'min': 1,
```
Comment (therobinkim) [inline faker.js]: the indentation looks pretty wonky here

File: client/src/index.jsx [hunk 6]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from '../components/App.jsx';
+
+ReactDOM.render(<App />, document.getElementById('root'));
```
Comment (therobinkim) [inline client/src/index.jsx]: put a newline at the end of each file

File: client/components/App.jsx [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    console.log('price', this.state.price)
+    return (
+      <div>
+          <Price price={this.state.price} />
+      </div>
+      )
+  }
+}
+
+export default App;
```
Comment (therobinkim) [inline client/components/App.jsx]: put a newline at the end of each file

File: client/components/App.jsx [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+  componentDidMount () {
+    this.getCars();
+  }
+
+  render () {
+    console.log('price', this.state.price)
+    return (
+      <div>
+          <Price price={this.state.price} />
```
Comment (therobinkim) [inline client/components/App.jsx]: ln 41 is indented too much

File: client/components/App.jsx [hunk 9]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Price from './Price.jsx';
+
+var server = "http://localhost:3000/api/turash/checkouts"
```
[the remark this answers is not in the document]
Comment (Arshavin86) [reply client/components/App.jsx]: I changed it on const. I'm planning to send POST requests to the server so I will use 'server' variable at list two times.

File: client/components/App.jsx (hunk 8 above)
Comment (Arshavin86) [reply client/components/App.jsx]: Fixed.

File: faker.js (hunk 5 above)
Comment (Arshavin86) [reply faker.js]: Fixed.

File: public/index.html (hunk 4 above)
Comment (Arshavin86) [reply public/index.html]: Fixed

File: webpack.config.js [hunk 10]
```diff
@@ -1,21 +1,23 @@
...
-    loaders : [
+  module: {
+    rules: [
       {
-        test : /\.jsx?/,
-        include : SRC_DIR,
-        loader : 'babel-loader',
-        query: {
-          presets: ['react', 'es2015']
+        test: /\.m?jsx$/,
```
[the remark this answers is not in the document]
Comment (Arshavin86) [reply webpack.config.js]: Changed on jsx

**Pull request merged**